types/pg/index.d.ts

```diff
@@ -1,6 +1,6 @@
-// Type definitions for pg 6.1
+// Type definitions for pg 7.1
```

Comment:
Could you back up version 6 to another folder as decsribed in the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-update-a-package-to-a-new-major-version)?

Reply:
I have copied the files prior to my change to folder `v6`.